src/frontend/src/apis/teams.api.ts

```diff
@@ -18,3 +18,9 @@ export const getSingleTeam = (id: string) => {
     transformResponse: (data) => JSON.parse(data)
   });
 };
+
+export const editSingleTeam = (id: string, payload: any) => {
```

Comment:
this should be `setTeamMembers` because we'll be having other endpoints that involve editing a tema like editing the description